test.sh

```diff
@@ -71,6 +71,10 @@ for version in "${meteor_versions[@]}"; do
   # Versions 2.2.2 need Node 12.22.4
   elif [[ "${version}" == 2.2.2 ]]; then
     node_version='12.22.4'
+
+  # Versions 2.2.3 need Node 12.22.5
+  elif [[ "${version}" == 2.2.3 ]]; then
+    node_version='12.22.5'
```

Comment:
Can you please clean up the whitespace in this file? It’s written using tabs.

Reply:
Sure.